frontend/components/Home/Dashboard/Dashboard.tsx

```diff
@@ -62,6 +60,40 @@ const Dashboard = () => {
     const [logOpen, setLogOpen] = useState(false);
     const [logModal, setLogModal] = useState(false);
 
+    const [messageDisp, setMessageDisp] = useState(false);
+   
+    const [tipCard, setTipCard] = useState({title: "", description: "", student: false});
```

Comment:
I was thinking about the design of this a bit more. Strictly, it's unnecessary for the tip to be stored as state. In general, if you can calculate a value on the fly,  you should do that rather than rely on state (this prevents inadvertent issues with inconsistent state or a blank tip card first loading and then populating with a tip). 

So the best approach is to define a function getTip() which would have the same logic as in your useEffect hook but then just return the tip message, and call that whenever you want to display the tip.